bin/get-geo-annotations.R

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env Rscript
+
```

Comment:
add this :) 
add this :) 
usePackage <- function(p) 
{
  if (!is.element(p, installed.packages()[,1]))
    install.packages(p, dep = TRUE)
  require(p, character.only = TRUE)
}

Reply:
@bswhite ?

Reply:
I've made @teslajoy 's two suggested changes above, @kkdang .